auditlog/tests/test_auditlog.py

```diff
@@ -321,6 +321,55 @@ def tearDown(self):
 
 
 class TestAuditlogFast(TransactionCase, AuditlogCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.test_model = (
```

Comment:
Do you need to setup a whole model? Can't you use a res.partner field like 'website'?

Reply:
Would it help to move the test (based on res.partner) into its own test class?